score/compute_score.py

```diff
@@ -0,0 +1,76 @@
+
+"""
+Compute the benchmark score given a frozen score configuration and current benchmark data.
+"""
+import argparse
+import json
+import math
+import os
+import yaml
+
+from tabulate import tabulate
+
+def compute_score(config, data, fake_data=None):
+    target = config['target']
+    score = 0.0
+    weight_sum = 0.0
+    for name in config['benchmarks']:
+        cfg = config['benchmarks'][name]
+        weight, norm = cfg['weight'], cfg['norm']
```

Comment:
It might also be good to check that weights are non-negative to avoid numerical instability in the previous calculations.

Reply:
yes, i was thinking of adding numerical stability check on all of [weight, norm, measurement].  I can catch weight >0 at the same time.  What do you think is a sensible epsilon value?  I think 1e-6 is fine for weight.  For measure and norm, the expected range could be anywhere from 1e3 to 1e9 i think, but we probably want to at least warn if not assert on norm / weight > 100 or something.